app/containers/HomeContainer/tests/index.test.js

```diff
@@ -52,6 +54,43 @@ describe('<HomeContainer /> tests', () => {
     renderProvider(<HomeContainer repoName={repoName} reposData={null} dispatchGithubRepos={submitSpy} />);
 
     await timeout(500);
-    expect(submitSpy).toBeCalled();
+    expect(submitSpy).toBeCalledWith(repoName);
```

Comment:
lets also please add some tests to make sure that the "repo-cards" are rendered. 


Let's please create another component called RepoCard that tests all of the intricacies of the RepoCard

Reply:
Sure , adding

Reply:
Updated accordingly